### DISCUSSION TITLE
Feature Request: Make the Statistics Display Summary positionable

### DISCUSSION THREAD

**Discussion by JSTCOFFEE** [artisan-roaster-scope/artisan#1025] [Ideas]
Where the Statistics Summary appears (top right of the display) often times obscures my view of the curve or temperature and time readings. The only way I can see of working around this is to adjust the axis to avoid this issue which makes a large portion of the screen useless. I think if the summary could be positioned on the screen in much the same way as the legend then that would solve this problem.
Apologies if this has already been addressed.
Many thanks.

**Comment by roasterdave** [comment]
The easiest thing is to tick **Auto** under Time Axis in the Config>> Axes dialog. This will automatically set your time axis to fit the graph with or without the Statistics Summary displayed. Turn Statistics Summary on and off and you will see how it adjusts. The Statistics Summary itself can be made narrower by reducing the **Max characters per line** setting in the Config>> Statistics dialog.

Thanks for the suggestion to make the Statistics Summary box draggable. I will put it on the list but it may be a while before it is considered for a change.
